types/entities.go

```diff
@@ -21,7 +21,10 @@ type Entity struct {
 }
 
 func (e Entities) MarshalJSON() ([]byte, error) {
-	s := e.toSlice()
+	s := maps.Values(e)
```

Comment:
Unrelated change. I just noticed that these could be inlined while I was looking for functions to export...